src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -860,16 +899,17 @@ private boolean isColonOfForEach(int parentType) {
     }
 
     /**
-     * Is array initialization.
+     * Is array initialization without whitespace around the array.
      *
      * @param currentType current token
      * @param parentType parent token
      * @return true is current token inside array initialization
+     *         and enforceWhitespaceAroundArrayInit is false.
      */
-    private static boolean isArrayInitialization(int currentType, int parentType) {
-        return currentType == TokenTypes.RCURLY
-                && (parentType == TokenTypes.ARRAY_INIT
-                        || parentType == TokenTypes.ANNOTATION_ARRAY_INIT);
+    private boolean isArrayInitWithoutWhitespaceAround(int currentType, int parentType) {
+        return !enforceWhitespaceAroundArrayInit
+                && currentType == TokenTypes.RCURLY
+                    && parentType == TokenTypes.ARRAY_INIT;
```

Comment:
implementation do not check Whitespace at all, please rename method.

Reply:
done